### PR TITLE
Gallery-page-fixed

### PR DESCRIPTION
**Pull request by kisharnath**
Gallery page not found fixed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,16 +1,16 @@
-**/*.db
```
Comment (picklelo) [inline .gitignore:1]: Can you undo the changes to this file?

**Review by picklelo — CHANGES_REQUESTED** [review]
Thanks for fixing - can you update the `.gitignore` changes?

File: .gitignore (hunk 1 above)
Comment (kisharnath) [reply .gitignore:1]: I have updated the file .

**Comment by picklelo** [comment]
This change made it in in another PR

**Pull request closed without merging**